web/src/runtime.ts

```diff
@@ -2473,6 +2473,7 @@ export class Instance implements Disposable {
     switch (tcode) {
       case ArgTypeCode.Int:
       case ArgTypeCode.UInt:
+      case ArgTypeCode.TVMArgBool:
         return this.memory.loadI64(rvaluePtr);
```

Comment:
cc @tqchen, would `loadI64()` be the right function to use for bool?

Reply:
this would need to wait until https://github.com/apache/tvm/pull/17240, but once the new bool type lands yes this would be the case